### PR TITLE
Add unit tests to strset

### PR DESCRIPTION
**Pull request by boaz0**
First commit :tada: :confetti_ball:  let me know if I missed anything.

Thanks :crossed_fingers: :pray:

### CODE REVIEW

File: strset/strset_test.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		t.Errorf("expected strSet %v to contain 'key1' but it does not", strSet)
+	}
+	if strSet.Contains("key2") == false {
+		t.Errorf("expected strSet %v to contain 'key2' but it does not", strSet)
+	}
+	if strSet.Len() != 2 {
+		t.Errorf("expected strSet %v to have length 2 but got %d", strSet, strSet.Len())
+	}
+
+	strSet.Remove("key0")
```
Comment (maskarb) [inline strset/strset_test.go:61]: Should we also check that this key does not exist above here?

File: strset/strset_test.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		t.Errorf("after removing 'key1' expected strSet %v to have length 1 but got %d", strSet, strSet.Len())
+	}
+}
+
+func TestRange(t *testing.T) {
+	tcs := []struct {
+		name string
+		keys []string
+	}{
+		{name: "strset contains 3 items", keys: []string{"ashley", "michael", "boaz"}},
```
Comment (maskarb) [inline strset/strset_test.go:89]: image

**Review by maskarb — APPROVED** [review]
Looks pretty good! Just one question, but otherwise approved.

File: strset/strset_test.go (hunk 1 above)
Comment (boaz0) [reply strset/strset_test.go:61]: Good point! Definitely yes

File: strset/strset_test.go (hunk 2 above)
Comment (boaz0) [reply strset/strset_test.go:92]: :100:

**Review by maskarb — APPROVED** [review]

**Comment by boaz0** [comment]
Thanks @maskarb

**Pull request merged**